### PR TITLE
Remove obsolete mobiletesting

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Ping @mbektchiev

### CODE REVIEW

File: sample-file-open-pdf/.abproject [hunk 1]
```diff
@@ -59,7 +59,6 @@
 	"Description": "sample-file-open-pdf",
 	"Framework": "Cordova",
 	"CordovaPluginVariables": {
-		"com.telerik.mobiletesting": {
```
Comment (mbektchiev) [inline sample-file-open-pdf/.abproject]: You should remove the whole object.

**Comment by mbektchiev** [comment]
Branch must be rebased

**Comment by Mitko-Kerezov** [comment]
Ping @mbektchiev with fixed comment and rebased branch

**Comment by mbektchiev** [comment]
:+!:

**Comment by mbektchiev** [comment]
:+1:

**Pull request merged**